cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+    for assessing the significance of *_min and *_max values,
+    and also for defining suitable contour levels.
+;
+    _name.category_id             refine_diff
+    _name.object_id               density_RMS
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            -100.:100.
+    _method.purpose               Definition
+    _method.expression
+;
+    # Default units. Applicable to X-ray and gamma radiation.
+    _units.code = "electrons_per_angstrom_cubed"
+
+    If (_diffrn_radiation.probe == "neutron")
+        _units.code = "femtometres_per_angstrom_cubed"
+
+    If (_diffrn_radiation.probe == "electron")
+        _units.code = "electrons_per_angstrom"
+;
+
+save_
+
+save_refine_diff.density_rms_su
+
+    _definition.id                '_refine_diff.density_RMS_su'
+
+    loop_
+      _alias.definition_id
+         '_refine_diff_density_RMS_su'
+         '_refine.diff_density_RMS_esd'
+
+    _definition.update            2024-05-01
+    _description.text
+;
+    Standard uncertainty of the root mean square density value
+    in a difference Fourier map.
+;
+    _name.category_id             refine_diff
+    _name.object_id               density_RMS_su
+    _name.linked_item_id          '_refine_diff.density_RMS'
+    _type.purpose                 SU
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Real
+    _method.purpose               Definition
+    _method.expression
+;
+    # Default units. Applicable to X-ray and gamma radiation.
+    _units.code = "electrons_per_angstrom_cubed"
+
+    If (_diffrn_radiation.probe == "neutron")
+        _units.code = "femtometres_per_angstrom_cubed"
+
+    If (_diffrn_radiation.probe == "electron")
+        _units.code = "electrons_per_angstrom"
+;
+
+save_
+
+save_refine_ls.abs_structure_z-score
+
+    _definition.id                '_refine_ls.abs_structure_z-score'
+    _alias.definition_id          '_refine_ls_abs_structure_z-score'
+
+    _definition.update            2024-05-03
+    _description.text
+;
+    The z-score is a measure of confidence in an absolute structure
+    determination as described by Klar et al. (2023), based on the
+    method of Le Page et al. (1990). For centrosymmetric structures,
+    the only permitted value, if the data item is present, is
+    'inapplicable', represented by '.' .
+    Ref:  Klar, P. B. et al. (2023). Nature Chem. 15, 848-855.
+          Le Page, Y., Gabe, E. J. & Gainsford, G. J. (1990). J. Appl. Cryst.
+          23, 406-411.
+;
+    _name.category_id             refine_ls
+    _name.object_id               abs_structure_z_score
+    _type.purpose                 Number
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   none
```

Comment:
Does the Z-score have any mathematical upper or lower limits (e.g. 0)?

Reply:
No; it's a statistical method that counts the number of Bijvoet pairs where the difference in intensity is a significant multiple of the standard deviation, so can increase according to the number of observations; but it can also be signed according as how one identifies the + and - sets of the pairs. And the DDLm checker tells me that

`_enumeration.range   :`

is an invalid construction.

Reply:
Ok, then this can be considered resolved.

`_enumeration.range    :` is indeed invalid since we expect either the lower bound, the upper bound or both. I understand that by ":" you want to express "no bounds", but currently this is done by not providing the _enumeration.range at all (as is done in the current definition so everything seems great).